### PR TITLE
guard new formatter behind env var

### PR DESCRIPTION
**Pull request by dnfield**
The new formatter isn't quite working in some scenarios.  Til I can get it straightened out it'll be disabled behind the env var `FLUTTER_TEST_FORMATTER`, which has to be set to the string `true` to re-enable it.

### CODE REVIEW

File: dev/bots/test.dart [hunk 1]
```diff
@@ -344,7 +350,7 @@ Future<void> _buildRunnerTest(
   bool enableFlutterToolAsserts = false,
   bq.TabledataResourceApi tableData,
 }) async {
-  final List<String> args = <String>['run', 'build_runner', 'test', '--', '-rcompact', '-j1'];
```
Comment (dnfield) [inline dev/bots/test.dart]: @jonahwilliams this _may_ have been the cause of the current failures, but I'lll try to figure that out for sure.

**Review by jonahwilliams — APPROVED** [review]
LGTM, thank you for taking care of this!

File: dev/bots/test.dart (hunk 1 above)
Comment (jonahwilliams) [reply dev/bots/test.dart]: Possible, but lets be a little conservative with these changes :)

Comment (dnfield) [reply dev/bots/test.dart]: Agreed.  We've got enough noise right now, I don't want to be adding to it.

**Pull request merged**